serine.galstyan/Lesson_6/exercise_12/exercise_12.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/evn python3
+
+collection_of_integers = []
+final_list = []
+index = 0
+entered_integer = input("Enter an integer: (blank to quit): ")
+while entered_integer != "":
+    collection_of_integers += [entered_integer]
```

Comment:
LW: This will separate decimal numbers. Try 12, 13, or other decimal numbers and see what happened after.